case-study-savas-labs.mdx

```diff
@@ -0,0 +1,86 @@
+---
+title: "Savas Labs saves 60+ hours in development time with Xata's Serverless Postgres"
+description: 'See how Savas Labs saved 60+ hours with Xata’s serverless Postgres solution, improving database management and accelerating digital projects for clients like the Environmental Integrity Project.'
+image:
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/sava-labs-case-study-og.png
+  alt: Case Study
+author: Cezzaine Zaher
+date: 09-05-2024
+tags: ['case-study']
+published: true
+slug: case-study-savas-labs
+ogImage: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/sava-labs-case-study-og@2x.png
+---
+
+![](/images/savas-labs-case-study-hero@2x.png)
+
+- **Company:** Savas Labs
+- **Industry:** Digital Agency
+- **Location:** Remote, HQ in Raleigh NC
+- **Size:** 30-35 employees
+
+[Savas Labs](https://savaslabs.com/), a full-service digital agency headquartered in North Carolina, is dedicated to delivering impactful and purpose-driven digital solutions. With a team of 30-35 professionals, including designers, developers, and project managers, Savas Labs has built a reputation for its focus on philanthropic work, particularly within academia and non-profit sectors. The agency has partnered with esteemed institutions like Duke University and Charlotte University. Their commitment to thorough discovery processes and meaningful client collaboration sets them apart from traditional agencies that often take a more generic approach to digital projects.
+
+## The task ahead
+
+Savas Labs was approached by the [Environmental Integrity Project](https://environmentalintegrity.org/) (EIP) to do a complete teardown and replacement of an existing Angular 1.0 application backed by a Postgres database. The purpose of the site is to show complicated environmental data (CO2e, economic effects, cancer risk, projected deaths, water crossings) of hundreds of facilities and pipelines throughout the United States, and display those in a way that is digestible to the user; be they an environmental scientist, legal professional, or curious citizen.
+
+## Challenges
+
+The existing [Oil & Gas Watch project](https://oilandgaswatch.org/) platform presented several challenges:
+
+- **Avoiding database management:** A The Savas Labs team wanted to avoid the complexities of managing a database infrastructure themselves.
+- **API accessibility:** A They needed a solution that could easily provide an API layer over the existing Postgres database without the need for extensive backend development.
+- **Data migration:** A Importing an existing Postgres database into a new system was a critical requirement.
+- **Geospatial data management:** A Savas Labs needed a solution to efficiently handle geospatial data with robust querying capabilities and integration with mapping tools. In addition, handling [Postgres branching](https://xata.io/branching) for different project environments posed unique challenges, which required Xata’s support.
```

Comment:
```suggestion
- **Geospatial data management:** Savas Labs needed a solution to efficiently handle geospatial data with robust querying capabilities and integration with mapping tools.
```

This felt a little out of place and I'm not sure if we necessarily need to call it out